pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -432,6 +432,9 @@ type ChildStatusReference struct {
 	runtime.TypeMeta `json:",inline"`
 	// Name is the name of the TaskRun or Run this is referencing.
 	Name string `json:"name,omitempty"`
+	// DisplayName is a user-facing name of the pipelineTask that may be
+	// used to populate a UI.
+	DisplayName string `json:"displayName,omitempty"`
```

Comment:
Does there need to be a conversion between v1beta1 and v1?

Reply:
Otherwise lgtm. Happy to approve as soon as this is resolved.

Reply:
thank you @chitrangpatel for the review, appreciate it! 🙏 

I am not sure about it. I have added the conversion [here](https://github.com/tektoncd/pipeline/compare/39daaf77ba155e10a994bb300855f0536205ef02..a9f4b619c0b57ea28c931c531bc872f65ad5893e), does it address the conversion? 🤔